index.html

```diff
@@ -168,6 +169,7 @@ <h3>BE A "NOBODY"</h3>
                         <a class="btn btn-large btn-clear" target="_blank" href="https://twitter.com/g0vtw"><i class="fui-twitter"></i> twitter</a>
                         <a class="btn btn-large btn-clear" target="_blank" href="https://www.facebook.com/g0v.tw/"><i class="fui-facebook"></i> Facebook</a>
                         <a class="btn btn-large btn-clear" target="_blank" href="https://github.com/g0v"><i class="fui-github"></i> github</a>

```

Comment:
Mind if we capitalize "GitHub" and "Twitter"?

Reply:
Yes that would be better

Reply:
:+1: Done and done